Proxy preferences do not work for HTTPS sites

## Version information

JabRef version JabRef 3.7-dev--snapshot--2016-11-07--master--f07fa32 windows 7 6.1 amd64 Java 1.8.0_112 on Windows 7

## Issue description

JabRef is no longer capable of getting through our authenticating firewall. I used to start JabRef like this:

`"C:\Program Files\Java\jre1.8.0_112\bin\javaw.exe" -Dhttp.proxyHost="<myproxy>" -Dhttp.proxyPort="8080" -jar "C:\Users\<myname>\AppData\Local\JabRef\JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar"`

This still worked with the 2016-10-16 snapshot but now no longer works with the 2016-11-07 snapshot. I also tried to specify proxy/port and user/password in JabRef's settings explicitly, which did not help either.

If possible, it would be great if you made the 2016-10-16 jar file available to me again (which was removed by the update) so that I can check that this older version still works with the proxy and that this issue is is not due to some changes on our side.

## Error console - messages caused by starting a medline search

```
10:56:46.754 [JabRef CachedThreadPool] ERROR net.sf.jabref.gui.importer.fetcher.SearchBasedEntryFetcher - Error while fetching from Medline
net.sf.jabref.logic.importer.FetcherException: Unable to get PubMed IDs
	at net.sf.jabref.logic.importer.fetcher.MedlineFetcher.getPubMedIdsFromQuery(MedlineFetcher.java:114) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.importer.fetcher.MedlineFetcher.performSearch(MedlineFetcher.java:162) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.gui.importer.fetcher.SearchBasedEntryFetcher.processQuery(SearchBasedEntryFetcher.java:37) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.gui.importer.fetcher.GeneralFetcher.lambda$actionPerformed$5(GeneralFetcher.java:217) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [?:1.8.0_112]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [?:1.8.0_112]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_112]
Caused by: java.net.UnknownHostException: eutils.ncbi.nlm.nih.gov
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source) ~[?:1.8.0_112]
	at java.net.PlainSocketImpl.connect(Unknown Source) ~[?:1.8.0_112]
	at java.net.SocksSocketImpl.connect(Unknown Source) ~[?:1.8.0_112]
	at java.net.Socket.connect(Unknown Source) ~[?:1.8.0_112]
	at sun.security.ssl.SSLSocketImpl.connect(Unknown Source) ~[?:1.8.0_112]
	at sun.security.ssl.BaseSSLSocketImpl.connect(Unknown Source) ~[?:1.8.0_112]
	at sun.net.NetworkClient.doConnect(Unknown Source) ~[?:1.8.0_112]
	at sun.net.www.http.HttpClient.openServer(Unknown Source) ~[?:1.8.0_112]
	at sun.net.www.http.HttpClient.openServer(Unknown Source) ~[?:1.8.0_112]
	at sun.net.www.protocol.https.HttpsClient.<init>(Unknown Source) ~[?:1.8.0_112]
	at sun.net.www.protocol.https.HttpsClient.New(Unknown Source) ~[?:1.8.0_112]
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.getNewHttpClient(Unknown Source) ~[?:1.8.0_112]
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(Unknown Source) ~[?:1.8.0_112]
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(Unknown Source) ~[?:1.8.0_112]
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(Unknown Source) ~[?:1.8.0_112]
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(Unknown Source) ~[?:1.8.0_112]
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(Unknown Source) ~[?:1.8.0_112]
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(Unknown Source) ~[?:1.8.0_112]
	at java.net.URL.openStream(Unknown Source) ~[?:1.8.0_112]
	at net.sf.jabref.logic.importer.fetcher.MedlineFetcher.getPubMedIdsFromQuery(MedlineFetcher.java:76) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	... 6 more
```

## Error console - messages at startup

By the way, I already get quite a number of warnings just after startup before doing anything in the application:

```
10:55:06.760 [AWT-EventQueue-0] INFO  net.sf.jabref.logic.importer.OpenDatabase - Opening: C:\local\Bibliography\bibliography.bib
10:55:07.906 [AWT-EventQueue-0] WARN  net.sf.jabref.model.search.rules.GrammarBasedSearchRule - Search query invalid
org.antlr.v4.runtime.misc.ParseCancellationException: null
	at org.antlr.v4.runtime.BailErrorStrategy.recoverInline(BailErrorStrategy.java:90) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at org.antlr.v4.runtime.Parser.match(Parser.java:227) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.search.SearchParser.start(SearchParser.java:110) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.model.search.rules.GrammarBasedSearchRule.init(GrammarBasedSearchRule.java:91) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.model.search.rules.GrammarBasedSearchRule.validateSearchStrings(GrammarBasedSearchRule.java:108) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.model.search.rules.SearchRules.getSearchRuleByQuery(SearchRules.java:18) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.model.search.GroupSearchQuery.getSearchRule(GroupSearchQuery.java:35) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.model.search.GroupSearchQuery.<init>(GroupSearchQuery.java:20) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.model.groups.SearchGroup.<init>(SearchGroup.java:42) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.groups.GroupsParser.searchGroupFromString(GroupsParser.java:168) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.groups.GroupsParser.fromString(GroupsParser.java:79) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.groups.GroupsParser.importGroups(GroupsParser.java:39) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.importer.util.MetaDataParser.parse(MetaDataParser.java:54) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.importer.fileformat.BibtexParser.parseFileContent(BibtexParser.java:237) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.importer.fileformat.BibtexParser.parse(BibtexParser.java:169) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.importer.fileformat.BibtexParser.parse(BibtexParser.java:89) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.importer.fileformat.BibtexImporter.importDatabase(BibtexImporter.java:70) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.importer.Importer.importDatabase(Importer.java:74) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.importer.fileformat.BibtexImporter.importDatabase(BibtexImporter.java:62) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.importer.OpenDatabase.loadDatabase(OpenDatabase.java:66) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.importer.OpenDatabase.loadDatabase(OpenDatabase.java:43) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.JabRefGUI.openLastEditedDatabases(JabRefGUI.java:213) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.JabRefGUI.openWindow(JabRefGUI.java:97) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.JabRefGUI.<init>(JabRefGUI.java:66) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.JabRefMain.lambda$start$1(JabRefMain.java:123) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at java.awt.event.InvocationEvent.dispatch(Unknown Source) [?:1.8.0_112]
	at java.awt.EventQueue.dispatchEventImpl(Unknown Source) [?:1.8.0_112]
	at java.awt.EventQueue.access$500(Unknown Source) [?:1.8.0_112]
	at java.awt.EventQueue$3.run(Unknown Source) [?:1.8.0_112]
	at java.awt.EventQueue$3.run(Unknown Source) [?:1.8.0_112]
	at java.security.AccessController.doPrivileged(Native Method) ~[?:1.8.0_112]
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source) [?:1.8.0_112]
	at java.awt.EventQueue.dispatchEvent(Unknown Source) [?:1.8.0_112]
	at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source) [?:1.8.0_112]
	at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source) [?:1.8.0_112]
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source) [?:1.8.0_112]
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source) [?:1.8.0_112]
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source) [?:1.8.0_112]
	at java.awt.EventDispatchThread.run(Unknown Source) [?:1.8.0_112]
Caused by: org.antlr.v4.runtime.InputMismatchException
	at org.antlr.v4.runtime.BailErrorStrategy.recoverInline(BailErrorStrategy.java:85) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	... 38 more
10:55:07.913 [AWT-EventQueue-0] WARN  net.sf.jabref.model.search.rules.GrammarBasedSearchRule - Search query invalid
org.antlr.v4.runtime.misc.ParseCancellationException: null
	at org.antlr.v4.runtime.BailErrorStrategy.recoverInline(BailErrorStrategy.java:90) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at org.antlr.v4.runtime.Parser.match(Parser.java:227) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.search.SearchParser.expression(SearchParser.java:220) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.search.SearchParser.start(SearchParser.java:108) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.model.search.rules.GrammarBasedSearchRule.init(GrammarBasedSearchRule.java:91) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.model.search.rules.GrammarBasedSearchRule.validateSearchStrings(GrammarBasedSearchRule.java:108) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.model.search.rules.SearchRules.getSearchRuleByQuery(SearchRules.java:18) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.model.search.GroupSearchQuery.getSearchRule(GroupSearchQuery.java:35) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.model.search.GroupSearchQuery.<init>(GroupSearchQuery.java:20) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.model.groups.SearchGroup.<init>(SearchGroup.java:42) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.groups.GroupsParser.searchGroupFromString(GroupsParser.java:168) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.groups.GroupsParser.fromString(GroupsParser.java:79) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.groups.GroupsParser.importGroups(GroupsParser.java:39) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.importer.util.MetaDataParser.parse(MetaDataParser.java:54) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.importer.fileformat.BibtexParser.parseFileContent(BibtexParser.java:237) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.importer.fileformat.BibtexParser.parse(BibtexParser.java:169) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.importer.fileformat.BibtexParser.parse(BibtexParser.java:89) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.importer.fileformat.BibtexImporter.importDatabase(BibtexImporter.java:70) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.importer.Importer.importDatabase(Importer.java:74) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.importer.fileformat.BibtexImporter.importDatabase(BibtexImporter.java:62) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.importer.OpenDatabase.loadDatabase(OpenDatabase.java:66) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.importer.OpenDatabase.loadDatabase(OpenDatabase.java:43) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.JabRefGUI.openLastEditedDatabases(JabRefGUI.java:213) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.JabRefGUI.openWindow(JabRefGUI.java:97) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.JabRefGUI.<init>(JabRefGUI.java:66) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.JabRefMain.lambda$start$1(JabRefMain.java:123) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at java.awt.event.InvocationEvent.dispatch(Unknown Source) [?:1.8.0_112]
	at java.awt.EventQueue.dispatchEventImpl(Unknown Source) [?:1.8.0_112]
	at java.awt.EventQueue.access$500(Unknown Source) [?:1.8.0_112]
	at java.awt.EventQueue$3.run(Unknown Source) [?:1.8.0_112]
	at java.awt.EventQueue$3.run(Unknown Source) [?:1.8.0_112]
	at java.security.AccessController.doPrivileged(Native Method) ~[?:1.8.0_112]
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source) [?:1.8.0_112]
	at java.awt.EventQueue.dispatchEvent(Unknown Source) [?:1.8.0_112]
	at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source) [?:1.8.0_112]
	at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source) [?:1.8.0_112]
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source) [?:1.8.0_112]
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source) [?:1.8.0_112]
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source) [?:1.8.0_112]
	at java.awt.EventDispatchThread.run(Unknown Source) [?:1.8.0_112]
Caused by: org.antlr.v4.runtime.InputMismatchException
	at org.antlr.v4.runtime.BailErrorStrategy.recoverInline(BailErrorStrategy.java:85) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	... 39 more
10:55:07.917 [AWT-EventQueue-0] WARN  net.sf.jabref.model.search.rules.GrammarBasedSearchRule - Search query invalid
org.antlr.v4.runtime.misc.ParseCancellationException: null
	at org.antlr.v4.runtime.BailErrorStrategy.recoverInline(BailErrorStrategy.java:90) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at org.antlr.v4.runtime.Parser.match(Parser.java:227) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.search.SearchParser.expression(SearchParser.java:220) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.search.SearchParser.start(SearchParser.java:108) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.model.search.rules.GrammarBasedSearchRule.init(GrammarBasedSearchRule.java:91) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.model.search.rules.GrammarBasedSearchRule.validateSearchStrings(GrammarBasedSearchRule.java:108) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.model.search.rules.SearchRules.getSearchRuleByQuery(SearchRules.java:18) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.model.search.GroupSearchQuery.getSearchRule(GroupSearchQuery.java:35) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.model.search.GroupSearchQuery.<init>(GroupSearchQuery.java:20) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.model.groups.SearchGroup.<init>(SearchGroup.java:42) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.groups.GroupsParser.searchGroupFromString(GroupsParser.java:168) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.groups.GroupsParser.fromString(GroupsParser.java:79) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.groups.GroupsParser.importGroups(GroupsParser.java:39) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.importer.util.MetaDataParser.parse(MetaDataParser.java:54) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.importer.fileformat.BibtexParser.parseFileContent(BibtexParser.java:237) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.importer.fileformat.BibtexParser.parse(BibtexParser.java:169) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.importer.fileformat.BibtexParser.parse(BibtexParser.java:89) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.importer.fileformat.BibtexImporter.importDatabase(BibtexImporter.java:70) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.importer.Importer.importDatabase(Importer.java:74) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.importer.fileformat.BibtexImporter.importDatabase(BibtexImporter.java:62) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.importer.OpenDatabase.loadDatabase(OpenDatabase.java:66) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.importer.OpenDatabase.loadDatabase(OpenDatabase.java:43) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.JabRefGUI.openLastEditedDatabases(JabRefGUI.java:213) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.JabRefGUI.openWindow(JabRefGUI.java:97) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.JabRefGUI.<init>(JabRefGUI.java:66) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.JabRefMain.lambda$start$1(JabRefMain.java:123) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at java.awt.event.InvocationEvent.dispatch(Unknown Source) [?:1.8.0_112]
	at java.awt.EventQueue.dispatchEventImpl(Unknown Source) [?:1.8.0_112]
	at java.awt.EventQueue.access$500(Unknown Source) [?:1.8.0_112]
	at java.awt.EventQueue$3.run(Unknown Source) [?:1.8.0_112]
	at java.awt.EventQueue$3.run(Unknown Source) [?:1.8.0_112]
	at java.security.AccessController.doPrivileged(Native Method) ~[?:1.8.0_112]
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source) [?:1.8.0_112]
	at java.awt.EventQueue.dispatchEvent(Unknown Source) [?:1.8.0_112]
	at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source) [?:1.8.0_112]
	at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source) [?:1.8.0_112]
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source) [?:1.8.0_112]
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source) [?:1.8.0_112]
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source) [?:1.8.0_112]
	at java.awt.EventDispatchThread.run(Unknown Source) [?:1.8.0_112]
Caused by: org.antlr.v4.runtime.InputMismatchException
	at org.antlr.v4.runtime.BailErrorStrategy.recoverInline(BailErrorStrategy.java:85) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	... 39 more
10:55:07.921 [AWT-EventQueue-0] WARN  net.sf.jabref.model.search.rules.GrammarBasedSearchRule - Search query invalid
org.antlr.v4.runtime.misc.ParseCancellationException: null
	at org.antlr.v4.runtime.BailErrorStrategy.recoverInline(BailErrorStrategy.java:90) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at org.antlr.v4.runtime.Parser.match(Parser.java:227) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.search.SearchParser.expression(SearchParser.java:220) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.search.SearchParser.start(SearchParser.java:108) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.model.search.rules.GrammarBasedSearchRule.init(GrammarBasedSearchRule.java:91) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.model.search.rules.GrammarBasedSearchRule.validateSearchStrings(GrammarBasedSearchRule.java:108) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.model.search.rules.SearchRules.getSearchRuleByQuery(SearchRules.java:18) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.model.search.GroupSearchQuery.getSearchRule(GroupSearchQuery.java:35) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.model.search.GroupSearchQuery.<init>(GroupSearchQuery.java:20) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.model.groups.SearchGroup.<init>(SearchGroup.java:42) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.groups.GroupsParser.searchGroupFromString(GroupsParser.java:168) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.groups.GroupsParser.fromString(GroupsParser.java:79) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.groups.GroupsParser.importGroups(GroupsParser.java:39) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.importer.util.MetaDataParser.parse(MetaDataParser.java:54) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.importer.fileformat.BibtexParser.parseFileContent(BibtexParser.java:237) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.importer.fileformat.BibtexParser.parse(BibtexParser.java:169) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.importer.fileformat.BibtexParser.parse(BibtexParser.java:89) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.importer.fileformat.BibtexImporter.importDatabase(BibtexImporter.java:70) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.importer.Importer.importDatabase(Importer.java:74) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.importer.fileformat.BibtexImporter.importDatabase(BibtexImporter.java:62) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.importer.OpenDatabase.loadDatabase(OpenDatabase.java:66) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.logic.importer.OpenDatabase.loadDatabase(OpenDatabase.java:43) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.JabRefGUI.openLastEditedDatabases(JabRefGUI.java:213) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.JabRefGUI.openWindow(JabRefGUI.java:97) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.JabRefGUI.<init>(JabRefGUI.java:66) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.JabRefMain.lambda$start$1(JabRefMain.java:123) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at java.awt.event.InvocationEvent.dispatch(Unknown Source) [?:1.8.0_112]
	at java.awt.EventQueue.dispatchEventImpl(Unknown Source) [?:1.8.0_112]
	at java.awt.EventQueue.access$500(Unknown Source) [?:1.8.0_112]
	at java.awt.EventQueue$3.run(Unknown Source) [?:1.8.0_112]
	at java.awt.EventQueue$3.run(Unknown Source) [?:1.8.0_112]
	at java.security.AccessController.doPrivileged(Native Method) ~[?:1.8.0_112]
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source) [?:1.8.0_112]
	at java.awt.EventQueue.dispatchEvent(Unknown Source) [?:1.8.0_112]
	at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source) [?:1.8.0_112]
	at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source) [?:1.8.0_112]
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source) [?:1.8.0_112]
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source) [?:1.8.0_112]
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source) [?:1.8.0_112]
	at java.awt.EventDispatchThread.run(Unknown Source) [?:1.8.0_112]
Caused by: org.antlr.v4.runtime.InputMismatchException
	at org.antlr.v4.runtime.BailErrorStrategy.recoverInline(BailErrorStrategy.java:85) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	... 39 more
10:55:07.926 [AWT-EventQueue-0] INFO  net.sf.jabref.logic.importer.OpenDatabase - Opening: C:\local\Bibliography\16349.bib
10:55:11.014 [SwingWorker-pool-5-thread-1] WARN  net.sf.jabref.gui.worker.VersionWorker - Could not connect to the updateserver.
java.net.UnknownHostException: api.github.com
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source) ~[?:1.8.0_112]
	at java.net.PlainSocketImpl.connect(Unknown Source) ~[?:1.8.0_112]
	at java.net.SocksSocketImpl.connect(Unknown Source) ~[?:1.8.0_112]
	at java.net.Socket.connect(Unknown Source) ~[?:1.8.0_112]
	at sun.security.ssl.SSLSocketImpl.connect(Unknown Source) ~[?:1.8.0_112]
	at sun.security.ssl.BaseSSLSocketImpl.connect(Unknown Source) ~[?:1.8.0_112]
	at sun.net.NetworkClient.doConnect(Unknown Source) ~[?:1.8.0_112]
	at sun.net.www.http.HttpClient.openServer(Unknown Source) ~[?:1.8.0_112]
	at sun.net.www.http.HttpClient.openServer(Unknown Source) ~[?:1.8.0_112]
	at sun.net.www.protocol.https.HttpsClient.<init>(Unknown Source) ~[?:1.8.0_112]
	at sun.net.www.protocol.https.HttpsClient.New(Unknown Source) ~[?:1.8.0_112]
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.getNewHttpClient(Unknown Source) ~[?:1.8.0_112]
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(Unknown Source) ~[?:1.8.0_112]
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(Unknown Source) ~[?:1.8.0_112]
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(Unknown Source) ~[?:1.8.0_112]
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(Unknown Source) ~[?:1.8.0_112]
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(Unknown Source) ~[?:1.8.0_112]
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(Unknown Source) ~[?:1.8.0_112]
	at net.sf.jabref.logic.util.Version.getAllAvailableVersions(Version.java:93) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.gui.worker.VersionWorker.doInBackground(VersionWorker.java:55) [JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at net.sf.jabref.gui.worker.VersionWorker.doInBackground(VersionWorker.java:29) [JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
	at javax.swing.SwingWorker$1.call(Unknown Source) [?:1.8.0_112]
	at java.util.concurrent.FutureTask.run(Unknown Source) [?:1.8.0_112]
	at javax.swing.SwingWorker.run(Unknown Source) [?:1.8.0_112]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [?:1.8.0_112]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [?:1.8.0_112]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_112]

```

Thanks for your report.!

Your error is related to the recent Medline changes. The fetcher now uses
https, and using an older version would be no help, as of Novemeber, 9th
2016.
https://www.ncbi.nlm.nih.gov/home/develop/https-guidance.shtml

You must now use the https.proxy.xxx arguments on startup:
http://www.glomelurus.com/2009/09/how-to-set-jvm-proxy-setting-without.html

2016-11-08 11:07 GMT+01:00 heseber <notifications@github.com>:

> Version information
>
> JabRef version JabRef 3.7-dev--snapshot--2016-11-07--master--f07fa32
> windows 7 6.1 amd64 Java 1.8.0_112 on Windows 7
> Issue description
>
> JabRef is no longer capable of getting through our authenticating
> firewall. I used to start JabRef like this:
>
> "C:\Program Files\Java\jre1.8.0_112\bin\javaw.exe"
> -Dhttp.proxyHost="<myproxy>" -Dhttp.proxyPort="8080" -jar
> "C:\Users<myname>\AppData\Local\JabRef\JabRef-3.7-dev--
> snapshot--2016-11-07--master--f07fa32.jar"
>
> This still worked with the 2016-10-16 snapshot but now no longer works
> with the 2016-11-07 snapshot. I also tried to specify proxy/port and
> user/password in JabRef's settings explicitly, which did not help either.
>
> If possible, it would be great if you made the 2016-10-16 jar file
> available to me again (which was removed by the update) so that I can check
> that this older version still works with the proxy and that this issue is
> is not due to some changes on our side.
> Error console - messages caused by starting a medline search
>
> 10:56:46.754 [JabRef CachedThreadPool] ERROR net.sf.jabref.gui.importer.fetcher.SearchBasedEntryFetcher - Error while fetching from Medline
> net.sf.jabref.logic.importer.FetcherException: Unable to get PubMed IDs
> 	at net.sf.jabref.logic.importer.fetcher.MedlineFetcher.getPubMedIdsFromQuery(MedlineFetcher.java:114) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.importer.fetcher.MedlineFetcher.performSearch(MedlineFetcher.java:162) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.gui.importer.fetcher.SearchBasedEntryFetcher.processQuery(SearchBasedEntryFetcher.java:37) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.gui.importer.fetcher.GeneralFetcher.lambda$actionPerformed$5(GeneralFetcher.java:217) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [?:1.8.0_112]
> 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [?:1.8.0_112]
> 	at java.lang.Thread.run(Unknown Source) [?:1.8.0_112]
> Caused by: java.net.UnknownHostException: eutils.ncbi.nlm.nih.gov
> 	at java.net.AbstractPlainSocketImpl.connect(Unknown Source) ~[?:1.8.0_112]
> 	at java.net.PlainSocketImpl.connect(Unknown Source) ~[?:1.8.0_112]
> 	at java.net.SocksSocketImpl.connect(Unknown Source) ~[?:1.8.0_112]
> 	at java.net.Socket.connect(Unknown Source) ~[?:1.8.0_112]
> 	at sun.security.ssl.SSLSocketImpl.connect(Unknown Source) ~[?:1.8.0_112]
> 	at sun.security.ssl.BaseSSLSocketImpl.connect(Unknown Source) ~[?:1.8.0_112]
> 	at sun.net.NetworkClient.doConnect(Unknown Source) ~[?:1.8.0_112]
> 	at sun.net.www.http.HttpClient.openServer(Unknown Source) ~[?:1.8.0_112]
> 	at sun.net.www.http.HttpClient.openServer(Unknown Source) ~[?:1.8.0_112]
> 	at sun.net.www.protocol.https.HttpsClient.<init>(Unknown Source) ~[?:1.8.0_112]
> 	at sun.net.www.protocol.https.HttpsClient.New(Unknown Source) ~[?:1.8.0_112]
> 	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.getNewHttpClient(Unknown Source) ~[?:1.8.0_112]
> 	at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(Unknown Source) ~[?:1.8.0_112]
> 	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(Unknown Source) ~[?:1.8.0_112]
> 	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(Unknown Source) ~[?:1.8.0_112]
> 	at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(Unknown Source) ~[?:1.8.0_112]
> 	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(Unknown Source) ~[?:1.8.0_112]
> 	at sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(Unknown Source) ~[?:1.8.0_112]
> 	at java.net.URL.openStream(Unknown Source) ~[?:1.8.0_112]
> 	at net.sf.jabref.logic.importer.fetcher.MedlineFetcher.getPubMedIdsFromQuery(MedlineFetcher.java:76) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	... 6 more
>
> Error console - messages at startup
>
> By the way, I already get quite a number of warnings just after startup
> before doing anything in the application:
>
> 10:55:06.760 [AWT-EventQueue-0] INFO  net.sf.jabref.logic.importer.OpenDatabase - Opening: C:\local\Bibliography\bibliography.bib
> 10:55:07.906 [AWT-EventQueue-0] WARN  net.sf.jabref.model.search.rules.GrammarBasedSearchRule - Search query invalid
> org.antlr.v4.runtime.misc.ParseCancellationException: null
> 	at org.antlr.v4.runtime.BailErrorStrategy.recoverInline(BailErrorStrategy.java:90) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at org.antlr.v4.runtime.Parser.match(Parser.java:227) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.search.SearchParser.start(SearchParser.java:110) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.model.search.rules.GrammarBasedSearchRule.init(GrammarBasedSearchRule.java:91) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.model.search.rules.GrammarBasedSearchRule.validateSearchStrings(GrammarBasedSearchRule.java:108) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.model.search.rules.SearchRules.getSearchRuleByQuery(SearchRules.java:18) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.model.search.GroupSearchQuery.getSearchRule(GroupSearchQuery.java:35) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.model.search.GroupSearchQuery.<init>(GroupSearchQuery.java:20) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.model.groups.SearchGroup.<init>(SearchGroup.java:42) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.groups.GroupsParser.searchGroupFromString(GroupsParser.java:168) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.groups.GroupsParser.fromString(GroupsParser.java:79) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.groups.GroupsParser.importGroups(GroupsParser.java:39) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.importer.util.MetaDataParser.parse(MetaDataParser.java:54) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.importer.fileformat.BibtexParser.parseFileContent(BibtexParser.java:237) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.importer.fileformat.BibtexParser.parse(BibtexParser.java:169) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.importer.fileformat.BibtexParser.parse(BibtexParser.java:89) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.importer.fileformat.BibtexImporter.importDatabase(BibtexImporter.java:70) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.importer.Importer.importDatabase(Importer.java:74) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.importer.fileformat.BibtexImporter.importDatabase(BibtexImporter.java:62) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.importer.OpenDatabase.loadDatabase(OpenDatabase.java:66) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.importer.OpenDatabase.loadDatabase(OpenDatabase.java:43) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.JabRefGUI.openLastEditedDatabases(JabRefGUI.java:213) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.JabRefGUI.openWindow(JabRefGUI.java:97) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.JabRefGUI.<init>(JabRefGUI.java:66) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.JabRefMain.lambda$start$1(JabRefMain.java:123) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at java.awt.event.InvocationEvent.dispatch(Unknown Source) [?:1.8.0_112]
> 	at java.awt.EventQueue.dispatchEventImpl(Unknown Source) [?:1.8.0_112]
> 	at java.awt.EventQueue.access$500(Unknown Source) [?:1.8.0_112]
> 	at java.awt.EventQueue$3.run(Unknown Source) [?:1.8.0_112]
> 	at java.awt.EventQueue$3.run(Unknown Source) [?:1.8.0_112]
> 	at java.security.AccessController.doPrivileged(Native Method) ~[?:1.8.0_112]
> 	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source) [?:1.8.0_112]
> 	at java.awt.EventQueue.dispatchEvent(Unknown Source) [?:1.8.0_112]
> 	at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source) [?:1.8.0_112]
> 	at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source) [?:1.8.0_112]
> 	at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source) [?:1.8.0_112]
> 	at java.awt.EventDispatchThread.pumpEvents(Unknown Source) [?:1.8.0_112]
> 	at java.awt.EventDispatchThread.pumpEvents(Unknown Source) [?:1.8.0_112]
> 	at java.awt.EventDispatchThread.run(Unknown Source) [?:1.8.0_112]
> Caused by: org.antlr.v4.runtime.InputMismatchException
> 	at org.antlr.v4.runtime.BailErrorStrategy.recoverInline(BailErrorStrategy.java:85) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	... 38 more
> 10:55:07.913 [AWT-EventQueue-0] WARN  net.sf.jabref.model.search.rules.GrammarBasedSearchRule - Search query invalid
> org.antlr.v4.runtime.misc.ParseCancellationException: null
> 	at org.antlr.v4.runtime.BailErrorStrategy.recoverInline(BailErrorStrategy.java:90) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at org.antlr.v4.runtime.Parser.match(Parser.java:227) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.search.SearchParser.expression(SearchParser.java:220) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.search.SearchParser.start(SearchParser.java:108) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.model.search.rules.GrammarBasedSearchRule.init(GrammarBasedSearchRule.java:91) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.model.search.rules.GrammarBasedSearchRule.validateSearchStrings(GrammarBasedSearchRule.java:108) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.model.search.rules.SearchRules.getSearchRuleByQuery(SearchRules.java:18) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.model.search.GroupSearchQuery.getSearchRule(GroupSearchQuery.java:35) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.model.search.GroupSearchQuery.<init>(GroupSearchQuery.java:20) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.model.groups.SearchGroup.<init>(SearchGroup.java:42) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.groups.GroupsParser.searchGroupFromString(GroupsParser.java:168) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.groups.GroupsParser.fromString(GroupsParser.java:79) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.groups.GroupsParser.importGroups(GroupsParser.java:39) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.importer.util.MetaDataParser.parse(MetaDataParser.java:54) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.importer.fileformat.BibtexParser.parseFileContent(BibtexParser.java:237) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.importer.fileformat.BibtexParser.parse(BibtexParser.java:169) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.importer.fileformat.BibtexParser.parse(BibtexParser.java:89) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.importer.fileformat.BibtexImporter.importDatabase(BibtexImporter.java:70) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.importer.Importer.importDatabase(Importer.java:74) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.importer.fileformat.BibtexImporter.importDatabase(BibtexImporter.java:62) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.importer.OpenDatabase.loadDatabase(OpenDatabase.java:66) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.importer.OpenDatabase.loadDatabase(OpenDatabase.java:43) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.JabRefGUI.openLastEditedDatabases(JabRefGUI.java:213) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.JabRefGUI.openWindow(JabRefGUI.java:97) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.JabRefGUI.<init>(JabRefGUI.java:66) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.JabRefMain.lambda$start$1(JabRefMain.java:123) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at java.awt.event.InvocationEvent.dispatch(Unknown Source) [?:1.8.0_112]
> 	at java.awt.EventQueue.dispatchEventImpl(Unknown Source) [?:1.8.0_112]
> 	at java.awt.EventQueue.access$500(Unknown Source) [?:1.8.0_112]
> 	at java.awt.EventQueue$3.run(Unknown Source) [?:1.8.0_112]
> 	at java.awt.EventQueue$3.run(Unknown Source) [?:1.8.0_112]
> 	at java.security.AccessController.doPrivileged(Native Method) ~[?:1.8.0_112]
> 	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source) [?:1.8.0_112]
> 	at java.awt.EventQueue.dispatchEvent(Unknown Source) [?:1.8.0_112]
> 	at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source) [?:1.8.0_112]
> 	at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source) [?:1.8.0_112]
> 	at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source) [?:1.8.0_112]
> 	at java.awt.EventDispatchThread.pumpEvents(Unknown Source) [?:1.8.0_112]
> 	at java.awt.EventDispatchThread.pumpEvents(Unknown Source) [?:1.8.0_112]
> 	at java.awt.EventDispatchThread.run(Unknown Source) [?:1.8.0_112]
> Caused by: org.antlr.v4.runtime.InputMismatchException
> 	at org.antlr.v4.runtime.BailErrorStrategy.recoverInline(BailErrorStrategy.java:85) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	... 39 more
> 10:55:07.917 [AWT-EventQueue-0] WARN  net.sf.jabref.model.search.rules.GrammarBasedSearchRule - Search query invalid
> org.antlr.v4.runtime.misc.ParseCancellationException: null
> 	at org.antlr.v4.runtime.BailErrorStrategy.recoverInline(BailErrorStrategy.java:90) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at org.antlr.v4.runtime.Parser.match(Parser.java:227) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.search.SearchParser.expression(SearchParser.java:220) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.search.SearchParser.start(SearchParser.java:108) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.model.search.rules.GrammarBasedSearchRule.init(GrammarBasedSearchRule.java:91) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.model.search.rules.GrammarBasedSearchRule.validateSearchStrings(GrammarBasedSearchRule.java:108) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.model.search.rules.SearchRules.getSearchRuleByQuery(SearchRules.java:18) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.model.search.GroupSearchQuery.getSearchRule(GroupSearchQuery.java:35) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.model.search.GroupSearchQuery.<init>(GroupSearchQuery.java:20) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.model.groups.SearchGroup.<init>(SearchGroup.java:42) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.groups.GroupsParser.searchGroupFromString(GroupsParser.java:168) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.groups.GroupsParser.fromString(GroupsParser.java:79) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.groups.GroupsParser.importGroups(GroupsParser.java:39) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.importer.util.MetaDataParser.parse(MetaDataParser.java:54) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.importer.fileformat.BibtexParser.parseFileContent(BibtexParser.java:237) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.importer.fileformat.BibtexParser.parse(BibtexParser.java:169) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.importer.fileformat.BibtexParser.parse(BibtexParser.java:89) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.importer.fileformat.BibtexImporter.importDatabase(BibtexImporter.java:70) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.importer.Importer.importDatabase(Importer.java:74) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.importer.fileformat.BibtexImporter.importDatabase(BibtexImporter.java:62) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.importer.OpenDatabase.loadDatabase(OpenDatabase.java:66) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.importer.OpenDatabase.loadDatabase(OpenDatabase.java:43) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.JabRefGUI.openLastEditedDatabases(JabRefGUI.java:213) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.JabRefGUI.openWindow(JabRefGUI.java:97) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.JabRefGUI.<init>(JabRefGUI.java:66) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.JabRefMain.lambda$start$1(JabRefMain.java:123) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at java.awt.event.InvocationEvent.dispatch(Unknown Source) [?:1.8.0_112]
> 	at java.awt.EventQueue.dispatchEventImpl(Unknown Source) [?:1.8.0_112]
> 	at java.awt.EventQueue.access$500(Unknown Source) [?:1.8.0_112]
> 	at java.awt.EventQueue$3.run(Unknown Source) [?:1.8.0_112]
> 	at java.awt.EventQueue$3.run(Unknown Source) [?:1.8.0_112]
> 	at java.security.AccessController.doPrivileged(Native Method) ~[?:1.8.0_112]
> 	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source) [?:1.8.0_112]
> 	at java.awt.EventQueue.dispatchEvent(Unknown Source) [?:1.8.0_112]
> 	at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source) [?:1.8.0_112]
> 	at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source) [?:1.8.0_112]
> 	at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source) [?:1.8.0_112]
> 	at java.awt.EventDispatchThread.pumpEvents(Unknown Source) [?:1.8.0_112]
> 	at java.awt.EventDispatchThread.pumpEvents(Unknown Source) [?:1.8.0_112]
> 	at java.awt.EventDispatchThread.run(Unknown Source) [?:1.8.0_112]
> Caused by: org.antlr.v4.runtime.InputMismatchException
> 	at org.antlr.v4.runtime.BailErrorStrategy.recoverInline(BailErrorStrategy.java:85) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	... 39 more
> 10:55:07.921 [AWT-EventQueue-0] WARN  net.sf.jabref.model.search.rules.GrammarBasedSearchRule - Search query invalid
> org.antlr.v4.runtime.misc.ParseCancellationException: null
> 	at org.antlr.v4.runtime.BailErrorStrategy.recoverInline(BailErrorStrategy.java:90) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at org.antlr.v4.runtime.Parser.match(Parser.java:227) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.search.SearchParser.expression(SearchParser.java:220) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.search.SearchParser.start(SearchParser.java:108) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.model.search.rules.GrammarBasedSearchRule.init(GrammarBasedSearchRule.java:91) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.model.search.rules.GrammarBasedSearchRule.validateSearchStrings(GrammarBasedSearchRule.java:108) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.model.search.rules.SearchRules.getSearchRuleByQuery(SearchRules.java:18) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.model.search.GroupSearchQuery.getSearchRule(GroupSearchQuery.java:35) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.model.search.GroupSearchQuery.<init>(GroupSearchQuery.java:20) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.model.groups.SearchGroup.<init>(SearchGroup.java:42) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.groups.GroupsParser.searchGroupFromString(GroupsParser.java:168) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.groups.GroupsParser.fromString(GroupsParser.java:79) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.groups.GroupsParser.importGroups(GroupsParser.java:39) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.importer.util.MetaDataParser.parse(MetaDataParser.java:54) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.importer.fileformat.BibtexParser.parseFileContent(BibtexParser.java:237) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.importer.fileformat.BibtexParser.parse(BibtexParser.java:169) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.importer.fileformat.BibtexParser.parse(BibtexParser.java:89) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.importer.fileformat.BibtexImporter.importDatabase(BibtexImporter.java:70) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.importer.Importer.importDatabase(Importer.java:74) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.importer.fileformat.BibtexImporter.importDatabase(BibtexImporter.java:62) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.importer.OpenDatabase.loadDatabase(OpenDatabase.java:66) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.logic.importer.OpenDatabase.loadDatabase(OpenDatabase.java:43) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.JabRefGUI.openLastEditedDatabases(JabRefGUI.java:213) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.JabRefGUI.openWindow(JabRefGUI.java:97) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.JabRefGUI.<init>(JabRefGUI.java:66) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.JabRefMain.lambda$start$1(JabRefMain.java:123) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at java.awt.event.InvocationEvent.dispatch(Unknown Source) [?:1.8.0_112]
> 	at java.awt.EventQueue.dispatchEventImpl(Unknown Source) [?:1.8.0_112]
> 	at java.awt.EventQueue.access$500(Unknown Source) [?:1.8.0_112]
> 	at java.awt.EventQueue$3.run(Unknown Source) [?:1.8.0_112]
> 	at java.awt.EventQueue$3.run(Unknown Source) [?:1.8.0_112]
> 	at java.security.AccessController.doPrivileged(Native Method) ~[?:1.8.0_112]
> 	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source) [?:1.8.0_112]
> 	at java.awt.EventQueue.dispatchEvent(Unknown Source) [?:1.8.0_112]
> 	at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source) [?:1.8.0_112]
> 	at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source) [?:1.8.0_112]
> 	at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source) [?:1.8.0_112]
> 	at java.awt.EventDispatchThread.pumpEvents(Unknown Source) [?:1.8.0_112]
> 	at java.awt.EventDispatchThread.pumpEvents(Unknown Source) [?:1.8.0_112]
> 	at java.awt.EventDispatchThread.run(Unknown Source) [?:1.8.0_112]
> Caused by: org.antlr.v4.runtime.InputMismatchException
> 	at org.antlr.v4.runtime.BailErrorStrategy.recoverInline(BailErrorStrategy.java:85) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	... 39 more
> 10:55:07.926 [AWT-EventQueue-0] INFO  net.sf.jabref.logic.importer.OpenDatabase - Opening: C:\local\Bibliography\16349.bib
> 10:55:11.014 [SwingWorker-pool-5-thread-1] WARN  net.sf.jabref.gui.worker.VersionWorker - Could not connect to the updateserver.
> java.net.UnknownHostException: api.github.com
> 	at java.net.AbstractPlainSocketImpl.connect(Unknown Source) ~[?:1.8.0_112]
> 	at java.net.PlainSocketImpl.connect(Unknown Source) ~[?:1.8.0_112]
> 	at java.net.SocksSocketImpl.connect(Unknown Source) ~[?:1.8.0_112]
> 	at java.net.Socket.connect(Unknown Source) ~[?:1.8.0_112]
> 	at sun.security.ssl.SSLSocketImpl.connect(Unknown Source) ~[?:1.8.0_112]
> 	at sun.security.ssl.BaseSSLSocketImpl.connect(Unknown Source) ~[?:1.8.0_112]
> 	at sun.net.NetworkClient.doConnect(Unknown Source) ~[?:1.8.0_112]
> 	at sun.net.www.http.HttpClient.openServer(Unknown Source) ~[?:1.8.0_112]
> 	at sun.net.www.http.HttpClient.openServer(Unknown Source) ~[?:1.8.0_112]
> 	at sun.net.www.protocol.https.HttpsClient.<init>(Unknown Source) ~[?:1.8.0_112]
> 	at sun.net.www.protocol.https.HttpsClient.New(Unknown Source) ~[?:1.8.0_112]
> 	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.getNewHttpClient(Unknown Source) ~[?:1.8.0_112]
> 	at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(Unknown Source) ~[?:1.8.0_112]
> 	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(Unknown Source) ~[?:1.8.0_112]
> 	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(Unknown Source) ~[?:1.8.0_112]
> 	at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(Unknown Source) ~[?:1.8.0_112]
> 	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(Unknown Source) ~[?:1.8.0_112]
> 	at sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(Unknown Source) ~[?:1.8.0_112]
> 	at net.sf.jabref.logic.util.Version.getAllAvailableVersions(Version.java:93) ~[JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.gui.worker.VersionWorker.doInBackground(VersionWorker.java:55) [JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at net.sf.jabref.gui.worker.VersionWorker.doInBackground(VersionWorker.java:29) [JabRef-3.7-dev--snapshot--2016-11-07--master--f07fa32.jar:?]
> 	at javax.swing.SwingWorker$1.call(Unknown Source) [?:1.8.0_112]
> 	at java.util.concurrent.FutureTask.run(Unknown Source) [?:1.8.0_112]
> 	at javax.swing.SwingWorker.run(Unknown Source) [?:1.8.0_112]
> 	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [?:1.8.0_112]
> 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [?:1.8.0_112]
> 	at java.lang.Thread.run(Unknown Source) [?:1.8.0_112]
>
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/JabRef/jabref/issues/2249>, or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AATi5GLmFY4PUH6gL2E5kIRM5DKx_gUiks5q8EnygaJpZM4KsPjI>
> .
>

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -25,6 +25,9 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 - We fixed issues with the [timestamp](http://help.jabref.org/en/TimeStamp) field. However, clearing with the clear button is not possible if timestamp format does not match the current settings. Fixes [#2403](https://github.com/JabRef/jabref/issues/2403).
 - Fixes [#2406](https://github.com/JabRef/jabref/issues/2406) so that the integrity check filter works again
 - Closing of subtrees in the groups panel using "close subtree" is working again. Fixes [#2319](https://github.com/JabRef/jabref/issues/2319).
+- The proxy settings are now also applied to HTTPS connections. Fixes (#2249)(https://github.com/JabRef/jabref/issues/2249).
+
+
 
 ### Removed
 

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -25,7 +25,7 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 - We fixed issues with the [timestamp](http://help.jabref.org/en/TimeStamp) field. However, clearing with the clear button is not possible if timestamp format does not match the current settings. Fixes [#2403](https://github.com/JabRef/jabref/issues/2403).
 - Fixes [#2406](https://github.com/JabRef/jabref/issues/2406) so that the integrity check filter works again
 - Closing of subtrees in the groups panel using "close subtree" is working again. Fixes [#2319](https://github.com/JabRef/jabref/issues/2319).
-- The proxy settings are now also applied to HTTPS connections. Fixes (#2249)(https://github.com/JabRef/jabref/issues/2249).
+- The proxy settings are now also applied to HTTPS connections. Fixes [#2249](https://github.com/JabRef/jabref/issues/2249).
 
 
 

--- a/src/main/java/net/sf/jabref/logic/net/ProxyRegisterer.java
+++ b/src/main/java/net/sf/jabref/logic/net/ProxyRegisterer.java
@@ -8,10 +8,16 @@ public class ProxyRegisterer {
             System.setProperty("http.proxyHost", proxyPrefs.getHostname());
             System.setProperty("http.proxyPort", proxyPrefs.getPort());
 
+            System.setProperty("https.proxyHost", proxyPrefs.getHostname());
+            System.setProperty("https.proxyPort", proxyPrefs.getPort());
+
             // NetworkTab.java ensures that proxyUsername and proxyPassword are neither null nor empty
             if (proxyPrefs.isUseAuthentication()) {
                 System.setProperty("http.proxyUser", proxyPrefs.getUsername());
                 System.setProperty("http.proxyPassword", proxyPrefs.getPassword());
+
+                System.setProperty("https.proxyUser", proxyPrefs.getUsername());
+                System.setProperty("https.proxyPassword", proxyPrefs.getPassword());
             }
         } else {
             // The following two lines signal that the system proxy settings

